4_advanced_usage/75_relay_servers.txt

```diff
@@ -25,134 +25,22 @@ The procedure will:
 
 ==== Installation procedure - On the relay
 
-===== Install Rudder agent
+===== Install Rudder server relay
 
-To begin, please install a regular Rudder agent on the OS.
-
-See <<install-agent, here>> for the complete reference about agent installation.
+To begin, please install a regular Rudder agent on the OS, following the
+<<install-agent, installation instructions>>, but install the 'rudder-server-relay'
+package in addition to the 'rudder-agent' package.
```

Comment:
rudder-server-relay should be enough but http://www.rudder-project.org/redmine/issues/7593.